implementations/ApiCatalog.json

```diff
@@ -1,95 +1,92 @@
 {
-	"name": "apicatalog.com",
-	"implementation": "Iron Verifiable Credentials",
-	"verifiers": [
-		{
-			"id": "https://apicatalog.com/vc/verifier",
-			"endpoint": "https://vc.apicatalog.com/credentials/verify",
-			"supportedEcdsaKeyTypes": [
-				"P-256",
-				"P-384"
-			],
-			"tags": [
-				"Ed25519Signature2020",
-				"vc2.0",
-				"eddsa-rdfc-2022",
-				"ecdsa-rdfc-2019",
-				"ecdsa-sd-2023"
-			]
-		}
-	],
-	"issuers": [
-		{
-			"id": "https://apicatalog.com/issuer/Ed25519Signature2020",
-			"endpoint": "https://vc.apicatalog.com/credentials/issue",
-			"options": {
-				"type": "Ed25519Signature2020"
-			},
-			"tags": [
-				"Ed25519Signature2020",
-				"vc2.0"
-			]
-		},
-		{
-			"id": "https://apicatalog.com/issuer/eddsa-rdfc-2022",
-			"endpoint": "https://vc.apicatalog.com/credentials/issue",
-			"options": {
-				"type": "eddsa-2022"
-			},
-			"tags": [
-				"eddsa-rdfc-2022"
-			]
-		},
-		{
-			"id": "https://apicatalog.com/issuer/ecdsa-rdfc-2019",
-			"endpoint": "https://vc.apicatalog.com/credentials/issue",
-			"options": {
-				"type": "ecdsa-2019"
-			},
-			"supportedEcdsaKeyTypes": [
-				"P-256",
-				"P-384"
-			],
-			"tags": [
-				"ecdsa-rdfc-2019"
-			]
-		},
-		{
-			"id": "https://apicatalog.com/issuer/ecdsa-sd-2023",
-			"endpoint": "https://vc.apicatalog.com/credentials/issue",
-			"options": {
-				"type": "ecdsa-sd-2023"
-			},
-			"supportedEcdsaKeyTypes": [
-				"P-256"
-			],
-			"tags": [
-				"ecdsa-sd-2023"
-			]
-		}
-	],
-	"vpVerifiers": [
-		{
-			"id": "https://apicatalog.com/vp/verifier",
-			"endpoint": "https://vc.apicatalog.com/presentations/verify",
-			"tags": [
-				"Ed25519Signature2020",
-				"vc2.0",
-				"eddsa-rdfc-2022",
-				"ecdsa-rdfc-2019",
-				"ecdsa-sd-2023"
-			]
-		}
-	],
-	"vcHolders": [
-		{
-			"id": "https://apicatalog.com/vp/holder",
-			"endpoint": "https://vc.apicatalog.com/credentials/derive",
-			"tags": [
-				"vcHolder",
-				"vc2.0",
-				"ecdsa-sd-2023"
-			]
-		}
-	]
+  "name": "apicatalog.com",
+  "implementation": "Iron Verifiable Credentials",
+  "verifiers": [
+    {
+      "id": "https://vc.apicatalog.com/verifier",
+      "endpoint": "https://vc.apicatalog.com/credentials/verify",
+      "supportedEcdsaKeyTypes": [
+        "P-256",
+        "P-384"
+      ],
+      "tags": [
+        "Ed25519Signature2020",
```

Comment:
I am going to go ahead and take this as I think this reveals an error in the suite and not necessarily one in your manifest. Technically speaking this tells the suite that this verifier endpoints supports `p-256` and `p-384` versions of all the signature suites specified in tags. Some of those permutations might not exist such as `ed25519Signatue2020` and `ecdsa-sd-2023`, so go ahead and feel free to merge this knowing I will be fixing some issues on the suites today so the suites know which keyTypes to test and only use the manifest to confirm that you support certain keyTypes.